express-pg-auth0/src/app.ts

```diff
@@ -134,5 +134,5 @@ export function create(env: Environment) {
     await destroyDbClient(dbClient);
   };
 
-  return { app, destroy };
+  return { app, dbClient, destroy };
```

Comment:
Looks weird the app to export a `dbClient` that's going to be used only in tests.
IMO we could either:
- have `destroy` to be sth like `async (client = dbClient) { ... }` and call it with the test `dbClient` when in tests
- in the tests files directly import the `destroy` method from the database module and have it destroy the test `dbClient` 
I'm not super convinced about this though so I'm putting it only for discussion :)

Reply:
Thanks, that makes sense :) 
But on the other hand, I don't think it is a big deal. Exporting the `dbClient` could be useful at some point, e.g. working on an implementation that requires direct reference to the `dbClient` object.

Just opened the discussion :D I am open to hearing others' opinions.